packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -309,20 +235,47 @@ DatepickerCalendar.propTypes = {
   today: PropTypes.instanceOf(Date),
   showWeekNumbers: PropTypes.bool,
   workWeekDays: PropTypes.arrayOf(PropTypes.oneOf(Object.keys(DayOfWeek).map(name => DayOfWeek[name]))),
-
-  format: PropTypes.func,
-  parse: PropTypes.func,
+  weeksToShow: PropTypes.number,
+
+  formatDay: PropTypes.func,
+  formatYear: PropTypes.func,
+  formatMonthDayYear: PropTypes.func,
+  formatMonthYear: PropTypes.func,
+
+  parseDate: PropTypes.func,
+
+  months: PropTypes.arrayOf(PropTypes.string),
+  shortMonths: PropTypes.arrayOf(PropTypes.string),
+  days: PropTypes.arrayOf(PropTypes.string),
+  shortDays: PropTypes.arrayOf(PropTypes.string),
+
+  isRequiredErrorMessage: PropTypes.string,
+  invalidInputErrorMessage: PropTypes.string,
+  isOutOfBoundsErrorMessage: PropTypes.string,
+  goToToday: PropTypes.string,
+  prevMonthAriaLabel: PropTypes.string,
+  nextMonthAriaLabel: PropTypes.string,
+  prevYearAriaLabel: PropTypes.string,
+  nextYearAriaLabel: PropTypes.string,
+  prevYearRangeAriaLabel: PropTypes.string,
+  nextYearRangeAriaLabel: PropTypes.string,
+  monthPickerHeaderAriaLabel: PropTypes.string,
+  yearPickerHeaderAriaLabel: PropTypes.string,
+  closeButtonAriaLabel: PropTypes.string,
+  weekNumberFormatString: PropTypes.string,
+  selectedDateFormatString: PropTypes.string,
+  todayDateFormatString: PropTypes.string,
 };
 
 DatepickerCalendar.defaultProps = {
   accessibility: datepickerCalendarBehavior,
-  firstDayOfWeek: DayOfWeek.Monday,
+  firstDayOfWeek: DayOfWeek.Sunday,
   firstWeekOfYear: FirstWeekOfYear.FirstDay,
   dateRangeType: DateRangeType.Day,
   header: {},
   calendarCell: {},
   calendarHeaderCell: {},
-  localizedStrings: DEFAULT_CALENDAR_LOCALIZED_STRINGS,
+  ...DEFAULT_CALENDAR_STRINGS,
```

Comment:
Please could you update DatepickerCalendarHeader.tsx with the localized string for next/previous month (there are TODO: marks)